module-system/sov-modules-stf-template/src/lib.rs

```diff
@@ -97,22 +115,28 @@ where
 
     type Witness = <<C as Spec>::Storage as Storage>::Witness;
 
-    fn init_chain(&mut self, params: Self::InitialState) {
+    type Condition = Cond;
+
+    fn init_chain(&mut self, params: Self::InitialState) -> anyhow::Result<jmt::RootHash> {
```

Comment:
What can return error here? I see that we panic, which is fine, but what is expected error case?

Maybe we should just `jmt::RootHash`?

Reply:
Yes indeed we can remove the result type here